scripts/custom/provision-50-provision-content.sh

```diff
@@ -23,8 +23,9 @@ fi
 
 if [ "${CIVICTHEME_GENERATED_CONTENT_CREATE_SKIP:-}" != "1" ]; then
   echo "  > Generate test content."
-  GENERATED_CONTENT_CREATE=1 drush pm-enable cs_generated_content
-
+  drush recipe /app/recipes/civictheme_content_generated_static
```

Comment:
This is contentious - but for now I want standardised generated content so we can visual regression (in a follow up PR).

Reply:
I've tested visual regression tool against this release to make sure everything looks the same visually